resources/views/scheduleInstrument.html

```diff
@@ -0,0 +1,427 @@
+                            element.addEventListener('mouseout', function() {
+                                $(element).popover('hide');
+                            });
+
+                        }
+                    });
+
+                    calendar.render();
+                });
+            }
+        });
+
+        LABKEY.Query.selectRows({
+            schemaName: 'targetedms',
+            queryName: 'paymentMethod',
+            sort: 'name',
+            columns: 'Id,UWBudgetNumber,worktag, budgetExpirationDate, name',
+            scope: this,
+            success: function (result) {
+                let rows = result.rows;
+                paymentMethodsData = rows;
+                let paymentMethodDropDown = '<p>Select Payment Method(s): <p> <br/>'+
+                        '<label for="paymentMethodDropDown">Payment Method:&nbsp</label>';
+                paymentMethodDropDown += '<select id="paymentMethodDropDown" name="paymentMethodDropDown">';
+                for (let i = 0; i < rows.length; i++) {
+                    paymentMethodDropDown += '<option value="' + LABKEY.Utils.encodeHtml(rows[i].Id) + '">' + LABKEY.Utils.encodeHtml(rows[i].name) + '</option>';
+                }
+                paymentMethodDropDown += '</select>';
+
+                paymentMethodDropDown += '&nbsp<input type="text" id="paymentMethodPercent" name="paymentMethodPercent" value="100" size="3" maxlength="3">';
+                paymentMethodDropDown += '<label for="paymentMethodPercent">%</label>';
+
+                jQuery('#paymentMethodDropDown').html(paymentMethodDropDown);
+
+            }
+        });
+
+        // LABKEY.Query.selectRows({
+        //     schemaName: 'targetedms',
+        //     queryName: 'instrumentOperator',
+        //     sort: 'name',
+        //     columns: 'Id,name',
+        //     scope: this,
+        //     success: function (result) {
+        //         let rows = result.rows;
+        //         let instrumentOperatorDropDown = '<label for="instrumentOperatorDropDown">Select Instrument Operator:</label>';
+        //         instrumentOperatorDropDown += '<select id="instrumentOperatorDropDown" name="instrumentOperatorDropDown">';
+        //         for (let i = 0; i < rows.length; i++) {
+        //             instrumentOperatorDropDown += '<option value="' + rows[i].Id + '">' + rows[i].name + '</option>';
+        //         }
+        //         instrumentOperatorDropDown += '</select>';
+        //         jQuery('#instrumentOperatorDropDown').html(instrumentOperatorDropDown);
+        //     }
+        // });
+
+        function zeroPadded(val) {
+            if (val >= 10)
+                return val;
+            else
+                return '0' + val;
+        }
+
+        function editEvent(event) {
+            let startDate = event.startDate;
+            let endDate = event.endDate;
+            let startDateVal = startDate.getFullYear() + "-" +
```

Comment:
Can this use the configured date/time too?

Reply:
done